docs/source/Quickstart/01_requirements.md

```diff
@@ -24,9 +24,9 @@ Your used must have full `sudo` access in order to:
 While we try to keep the footprint low on the system, packages are needed, and
 some 3rd-party software require accesses, such as CRC.
 
-The best way to get that is to push this content to `/etc/suders.d/USERNAME`:
+The best way to get that is to push this content to `/etc/sudoers.d/USERNAME`:
```

Comment:
I'm gonna suggest, to explicitly told the user to make this change using `visudo` just to avoid someone getting locked in his on machine if the modification is wrongly spelled/copy-pasted.